### PR TITLE
Mann whitney testing

### PR DESCRIPTION
**Pull request by galynazholtkevych**

### CODE REVIEW

**Comment by ometelytsia** [comment]
Need to change target to `dev`.

File: app/reports_generation/performance_profile.yml [hunk 1]
```diff
@@ -3,12 +3,15 @@ column_name: "90% Line"
...
+    fullPath: "" # abs path of second run ".../dc-app-performance-toolkit/app/results/reports/2022-01-06_14-47-01"
 # Chart generation config
 index_col: "Action"
 title: "DCAPT Performance Testing"
 image_height_px: 1000
-image_width_px: 1600
\ No newline at end of file
+image_width_px: 1600
+judge: false # enable for judgement
+send_report: false
```
Comment (ometelytsia) [inline app/reports_generation/performance_profile.yml:17]: let's remove `send_report` for now

**Review by SergeyMoroz0703 — DISMISSED** [review]

File: app/reports_generation/csv_chart_generator.py [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import datetime
 from pathlib import Path
 
-from scripts import config_provider, csv_aggregator, chart_generator, summary_aggregator, results_archivator
+from reports_generation.scripts import (config_provider, csv_aggregator, chart_generator,
```
Comment (ometelytsia) [inline app/reports_generation/csv_chart_generator.py:4]: Could you pls make a script to work from the `app/reports_generation` folder as this is described in the documentation.

File: app/util/data_preparation/dataframe_converter.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
+from decimal import Decimal, getcontext
```
Comment (ometelytsia) [inline app/util/data_preparation/dataframe_converter.py:1]: Can we move this file from this folder to`app/reports_generation/scipts`? 
This should simplify imports and makes all related to judgment code in one place.

**Pull request closed without merging**